zerver/lib/actions.py

```diff
@@ -2682,6 +2715,8 @@ def fetch_initial_state_data(user_profile, event_types, queue_id):
 
     if want('stream'):
         state['streams'] = do_get_streams(user_profile)
+    if want('update_default_streams'):
+        state['realm_default_streams'] = json_serialize_streams(get_default_streams_for_realm(user_profile.realm))
```

Comment:
@timabbott Updated code. Is there any simpler way to do this without the json_serialize_streams function?